pages/common/dirs.md

```diff
@@ -18,3 +18,7 @@
 - Clear the directory stack:
 
 `dirs -c`
+
+- Show indexes:
```

Comment:
Perhaps this would read better to match the format of other examples:
```suggestion
- Display the indexes:
```

Reply:
I think the correct word is indices.